docs/proposals/issue-priorities.md

```diff
@@ -0,0 +1,6 @@
+These are the meanings of the labels priority/p0 ... priority/p3 that we apply to issues in order to try to prioritize them relative to each other.  We try to apply these priority labels consistently across the entire project, but if you notice an issue that you believe to be misprioritized, please do let us know and we will evaluate your counter-proposal.
+
+- **priority/P0**: Must be actively worked on as someone's top priority right now. Stuff is burning. If it's not being actively worked on, someone is expected to drop what they're doing immediately to work on it. TL's of teams are responsible for making sure that all P0's in their area are being actively worked on.
```

Comment:
Examples: user-visible bug in core features, broken build or tests, critical security issue.